lib/main.dart

```diff
@@ -58,6 +59,7 @@ main(List<String> args) async {
   Hive.init(appDir.path);
   Hive.registerAdapter<MemberModel>(MemberAdapter());
   Hive.registerAdapter<TransactionModel>(TransactionAdapter());
+  PushNotificationService().initialise();
```

Comment:
Init the Push notification service singleton

Reply:
This is the best place i could find to Initialize this service. 
If there is a better place, let me know.